applications/ecallmgr/src/ecallmgr_call_command.erl

```diff
@@ -110,6 +110,13 @@ get_fs_app(_Node, _UUID, JObj, <<"hangup">>) ->
         'true' ->  {<<"unbridge">>, <<>>}
     end;
 
+get_fs_app(_Node, UUID, JObj, <<"audio_level">>) ->
+    Action = wh_json:get_value(<<"Action">>, JObj),
+    Level = wh_json:get_value(<<"Level">>, JObj),
+    Mode = wh_json:get_value(<<"Mode">>, JObj),
+    Data = <<UUID/binary, " ", Action/binary, " ", Mode/binary, " mute ", Level/binary>>,
```

Comment:
I'm going to run this by the FreeSWITCH team to see if they have a better way to accomplish this. I don't fully understand the `uuid_audio` functionality, so can't comment yet if there's a better approach. If they don't have suggestions, we'll merge this in today.